internal/api/syncFlags.go

```diff
@@ -60,33 +70,18 @@ func SendFlagsToAPI(results []model.FileSearchResult, envId string) (err error)
 	}
 	flagUsageRequest.Flags = flags
 
-	err = callAPI(envId, flagUsageRequest)
+	err = callAPI(cfg, flagUsageRequest)
 
 	return err
 }
 
-func generateAuthenticationToken() error {
-	if os.Getenv("FLAGSHIP_CLIENT_ID") == "" {
-		log.WithFields(log.Fields{"variable": "FLAGSHIP_CLIENT_ID"}).Fatal("Missing required environment variable")
-	}
-
-	if os.Getenv("FLAGSHIP_CLIENT_SECRET") == "" {
```

Comment:
shouldn't be a validation logic here for cfg.FlagshipClientID like in previous os.GetEnv("Flagship_client_ID")